paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetContract.kt

```diff
@@ -38,14 +38,31 @@ class PaymentSheetContract :
     }
 
     @Parcelize
-    data class Args @VisibleForTesting internal constructor(
+    data class Args internal constructor(
         internal val clientSecret: ClientSecret,
         internal val config: PaymentSheet.Configuration?,
         @ColorInt internal val statusBarColor: Int? = null,
         @InjectorKey internal val injectorKey: String = DUMMY_INJECTOR_KEY
     ) : ActivityStarter.Args {
+
         val googlePayConfig: PaymentSheet.GooglePayConfiguration? get() = config?.googlePay
 
+        internal fun toV2(): PaymentSheetContractV2.Args {
+            return PaymentSheetContractV2.Args(
```

Comment:
The temporary bridge between `PaymentSheetContract` and `PaymentSheetContractV2`.